src/templates/CentralTemplateProvider.ts

```diff
@@ -96,6 +97,22 @@ export class CentralTemplateProvider implements Disposable {
         }
     }
 
+
+    public async getFunctionV2Templates(context: IActionContext, projectPath: string | undefined, language: ProjectLanguage, languageModel: number | undefined, version: FuncVersion, templateFilter: TemplateFilter, projectTemplateKey: string | undefined): Promise<FunctionV2Template[]> {
+        const templates: ITemplates = await this.getTemplates(context, projectPath, language, languageModel, version, projectTemplateKey);
+        switch (templateFilter) {
+            case TemplateFilter.All:
+                return templates.functionTemplatesV2;
+            // TODO: Verify if schema V2 templates have categories
```

Comment:
status on this todo?

Reply:
They do not. We could still have All vs. Verified (once we add tests) but I'm not sure if there's much value in that. I may just remove the filter for V2